bin/run_umt.sh

```diff
@@ -45,44 +45,33 @@ if [ "$1" == "build_umt" ]; then
     pushd ${UMT_PATH}
     mkdir -p ${UMT_INSTALL_PATH}
     pushd ${UMT_PATH}/umt_workspace
-    echo "*** Fetch and build dependencies ***"
 
-    git clone https://github.com/LLNL/UMT_TPLS.git
-
-    tar xvzf UMT_TPLS/metis-5.1.0.tar.gz
-    pushd metis-5.1.0
-    cmake . -DCMAKE_INSTALL_PREFIX=${UMT_INSTALL_PATH} -DCMAKE_C_COMPILER=${CC} -DCMAKE_CXX_COMPILER=${CXX} -DGKLIB_PATH=${PWD}/GKlib
-    make -j install
-    popd
-
-    tar xvzf UMT_TPLS/conduit-v0.8.8-src-with-blt.tar.gz
+    git clone --recurse-submodules  https://github.com/LLNL/conduit.git conduit -b v0.9.0
```

Comment:
Depending on size and download speed, we can consider using `--single-branch` and `--depth 1` as well?
Not sure how well that works with the submodules though.